2. Web API now calls Microsoft Graph/TodoListService/appsettings.json

```diff
@@ -20,7 +21,7 @@
           App ID URI of a legacy v1 web application
         Applications are registered in the https://portal.azure.com portal.
       */
-      "BaseUrl": "[WebApiUrl]",
+      "BaseUrl": "https://graph.microsoft.com/v1.0",
```

Comment:
Why let customers guess this? 
It's better to provide it in the appsettings.json.